source/documentation/05-payment-overview.md

```diff
@@ -105,21 +108,57 @@ If the details are valid and the payment is approved, the user is then taken to
 
 ![](https://s3-eu-west-1.amazonaws.com/pay-govuk-documentation/flow-payment-confirm-page.png)
 
-After confirming, the user is directed to the ``return_url`` you provided in the initial request.
+The user clicks confirm, and what happens next depends on whether the payment was successful or not.
 
-If the payment *cannot* be approved, the user is shown an error page with a message describing the reason, for example:
+##### Successful Payment
 
-![](https://s3-eu-west-1.amazonaws.com/pay-govuk-documentation/flow-payment-declined.png)
+If the payment is successful, the user will receive a confirmation email (if you have chosen to send these using GOV.UK Pay), and return to your service and see your confirmation page.
```

Comment:
Do we need to make clear that they need to show their successful payment page, rather than implying it will happen automatically?